package-lock.json

```diff
@@ -0,0 +1,17914 @@
+{
+  "name": "name-examination-upgrade",
+  "lockfileVersion": 3,
```

Comment:
What version of Node are you using? This should be version 2 (created by Node v16.14.2 or maybe later).

Reply:
I'm using v16.14.2

Reply:
Maybe the lock file was created with a later version of Node. Try deleting it and then `npm i`.